src/screens/credits.rs

```diff
@@ -5,27 +5,17 @@ use bevy::prelude::*;
 use super::Screen;
 use crate::{
     game::{assets::SoundtrackKey, audio::soundtrack::PlaySoundtrack},
-    ui::prelude::*,
+    ui::{interaction::OnPress, prelude::*},
```

Comment:
I'd add `OnPress` to the `ui` prelude.

Reply:
[`c81067a` (#229)](https://github.com/TheBevyFlock/bevy_quickstart/pull/229/commits/c81067a2cfe7a7faef579bfa319c75f65d696aeb)